### ISSUE TITLE
Add Rest & Soap Api

### ISSUE THREAD

**Issue by freebird1195** [freeCodeCamp/guide#4290]
Add an article on Rest & Soap Api

**Comment by KaraAJC** [comment]
I just read a great article today about this! 
Where does this get posted?

**Comment by andres-uris** [comment]
@KaraAJC  I want to help with the SOAP example 😄

**Comment by KaraAJC** [comment]
woohoo!! Here is the article I'm going to refer to in the guide: 

![soap v rest difference infographic](https://content-static.upwork.com/blog/uploads/sites/3/2017/04/19075723/SOAP-v-REST-1.png)

[SOAP & REST | Comparing two API architectures by Carey Wodehouse](https://www.upwork.com/hiring/development/soap-vs-rest-comparing-two-apis/)

**Comment by andres-uris** [comment]
AWESOMEE 👍

**Comment by KaraAJC** [comment]
@freebird1195 @ChaituVR 
Would creating the article inside `guides/src/pages/miscellaneous/` make sense?

**Comment by ChaituVR** [comment]
Makes Sense  :+1: But before starting the article consider looking into this PR #2908

can be useful if this PR is merged in the Future to avoid duplication 

Thanks @KaraAJC

**Comment by KaraAJC** [comment]
Thank you!! @ChaituVR! 
now that I'm preparing to build this article, I realize it might be better to create a section called **`WEB`** that covers all kinds of web-specific things, like:
- what an API is, 
- what is a Web Service, 
- REST, 
- SOAP 
- what webAPIs exist.. 
- HTTP... 

I can create a new issue for that, and just focus on this article, which describes REST vs SOAP...
OR I can just create this article inside a folder called web, and leave the article re-organization to y'all.

**Comment by ChaituVR** [comment]
It is a good idea to create a new folder named `web` but if it is only one article for now then it is good to go with `miscellaneous` folder itself and move it later when you get more articles 

@Bouncey @texas2010 any thoughts?

**Comment by KaraAJC** [comment]
I think the REST api article you referred to would make sense there, and I'd be willing to pivot to have @andres-uris just write a SOAP article, and I'll write an HTTP protocol article instead, giving the `web` folder 3 completed articles, out of an anticipated 7: 

Here's a [commit I made](https://github.com/KaraAJC/guides/commit/a810b9a49cc93fe3369e364509438c4719d9866a) so far: 
![screen shot 2017-10-27 at 9 54 15 am](https://user-images.githubusercontent.com/7103652/32110351-eede9a6a-bafc-11e7-97d4-0f63666629a9.png)

**Comment by andres-uris** [comment]
@KaraAJC Ok sounds good, I will write SOAP article on src/pages/web/soap/index.md . Sorry for the delay answer.
